src/regressionmodel.jl

```diff
@@ -34,19 +34,25 @@ Return the mean of the response.
 function meanresponse end
 
 """
-    modelmatrix(model::RegressionModel)
+    modelmatrix(model::RegressionModel; weighted::Bool=false)
 
-Return the model matrix (a.k.a. the design matrix).
+Return the model matrix (a.k.a. the design matrix) or, if `weighted=true` the weighted 
+model matrix, i.e. `X' * sqrt.(W)`, where `X` is the model matrix and
```

Comment:
Why transpose `X`? It sounds weird to change the orientation of the result depending on whether it's weighted or not.

Reply:
My bad...I will fix it